great-idea/css/index.css

```diff
@@ -144,87 +231,103 @@ footer {
 	text-align: center;
 }
 
-.services-nav{
-	margin:20px;
+.services-nav {
+	margin: 20px;
 }
 
```

Comment:
best practices here would be to copy the nav over from the index.html and paste it in every other page on the site and use just one set of styling rules that would apply the same to each nav so they all would be exactly the same on every page.